public/README.md

```diff
@@ -0,0 +1,7 @@
+Web App: Practice Place
+Creators: Alejandro, Matt, Zach
+Team Name: Team PP
+
+Practice Place is a web app Team PP created to solve the problem domain of surplus of musicians seeking safe and affordable rehearsal space.
+
+The App works by loading the site, navigating to the search page by clicking "Find now". Then the user enters the desired location the band wishes to practice in. After a comparison function a new page loads with available spaces displayed according to the location preference. The User can then sort through the locations, seeing the different properties of teach one. Once a location is picked the user can then reach out via the email provided to more forward with the renting process.
```

Comment:
Line 7 type "teach" --> "each" and maybe just delete the "more" in the sentence "provided to more forward with the renting process." Other than that looks good.